doc/devel/document.rst

```diff
@@ -194,9 +194,25 @@ nor the ````literal```` role:
 Refer to other documents and sections
 -------------------------------------
 
-Sphinx_ allows internal references_ between documents.
+Sphinx_ supports internal references_:
 
-Documents can be linked with the ``:doc:`` directive:
+==========  ===========================  ===========================================
+Role        Links target                  Representation in rendered HTML
+==========  ===========================  ===========================================
+|doc-dir|_  document                      link to a page
+|ref-dir|_  reference label (associated  link to an anchor associated with a heading
+            with a section)
```

Comment:
Usually, I'm not a fan of parantheses here as well.

Motivation: The technical reference point is the label, but semantically we refer to the associated section. I wanted to
- have the focus on the label, because that's what you have to specify in the role
- also give the context that the label represents a section

The parantheses should lower the importance of the context information wrt to the label itself.

Reply:
You are right. I've removed the parenthesis including its content.